app/javascript/react/components/FlavorShowPage.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import FlavorInformationComponent from './FlavorInformationComponent';
+
+const FlavorShowPage = (props) => {
+  const [getFlavorData, setFlavorData] = useState({
+    name: "",
+    description: "",
+    image_url: ""
+  });
+
+  useEffect(() => {
+  let region_id = props.match.params.region_id
+  let flavor_id = props.match.params.id
+    fetch(`/api/v1/regions/${region_id}/flavors/${flavor_id}`)
```

Comment:
Lets update indentation on lines 12-13

unless you need the region id for something, you likely can unnest this fetch request as well

Reply:
Made suggested changes